README.md

```diff
@@ -204,26 +204,33 @@ design your app with Taiga UI components.
                         src="https://raw.githubusercontent.com/MarsiBarsi/readme-icons/main/send.svg"
                 /></a>
             </div>
-        </td>
+        </td> 
         <td align="center">
-            <a href="https://github.com/mdlufy"
```

Comment:
semantically, has anything changed?

Reply:
I don't think the order matters, I make less impact and work part time in a team, so the order can be left as is :)